.envrc.sample

```diff
@@ -7,6 +7,3 @@ export REACT_APP_API_ENDPOINT="http://localhost:3001/submit-response"
 # allow fetching data from the open data site dev version
 export REACT_APP_DATA_ENDPOINT=https://data.dev.oiretutka.fi/
 export REACT_APP_DATA_AUTH=symptomradar:SECRET
-
-# Check and use required Node version
```

Comment:
Good thing that these were removed from the sample! I'm using `autoenv` and `nvm` commands fail bad on that one. BTW, these scripts look oddly familiar 🤨

Reply:
Also, should we add the backend environment variables to the sample? There's currently 7 envars for the code, 3 for AWS and 3 for Terraform. I think some awareness should be made about these environment variables.

Reply:
Hmm, yes.

There IS something already that makes the backend developers life a bit easier, though:

https://github.com/futurice/symptomradar/blob/ea168bb9861ce3d400c612953e293855482ee916/infra/modules/main/outputs.tf#L21-L24

So if you go:

```
$ terraform output
env_dev = {
  ...
  "lambda_env" = {
    "ABUSE_DETECTION_TABLE" = "symptomradar-dev-abuse-detection"
    "ATHENA_DB_NAME" = "symptomradar_dev_storage"
    "BUCKET_NAME_ATHENA_RESULTS" = "symptomradar-dev-storage-results"
    "BUCKET_NAME_OPEN_DATA" = "symptomradar-dev-open-data"
    "BUCKET_NAME_STORAGE" = "symptomradar-dev-storage"
    "CORS_ALLOW_ORIGIN" = "*"
    "DOMAIN_NAME_OPEN_DATA" = "data.dev.oiretutka.fi"
    "KNOWN_HASHING_PEPPER" = "D4GxgVVh0XQCrVb7FiyCal5ZgnRVkiVf"
    "SSM_SECRETS_PREFIX" = "/symptomradar/dev/"
  }
  ...
}
env_prod = {
  ...
}
```

...you can just prefix those with `export ` and you have an env that's guaranteed to equal the one on Lambda (because it's the same config that Terraform pushes to the backend).